1-JavaScript/Week3/homework/ex5-shoppingCartRedux.js

```diff
@@ -12,8 +12,12 @@ it pure. Do the following:
 4. When constructing the new shopping cart array you should make use of the ES5 
    spread syntax.
 ------------------------------------------------------------------------------*/
-function addToShoppingCart(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+function addToShoppingCart(shoppingCart, groceryItem) {// TODO complete this function
```

Comment:
good job!

Another way to do it would be without using a temporary variable:

```
function addToShoppingCart(arr, strGrocery) {
  // The cart length is lower than 3 so we return a new array adding a new item
  if (arr.length < 3) {
       return [...arr, strGrocery]
  }
 
  // The cart length is equals or greater than 3 so we return a new array extracting
  // the first item and adding the new item
  return [...arr.slice(1), strGrocery];
}
```